docs/general/administration/hardware-acceleration/index.md

```diff
@@ -180,13 +180,13 @@ Dolby Vision (P5 & P8) to SDR tone-mapping is supported in Jellyfin 10.8 and req
 
 :::
 
-| OS/Platform    | NVIDIA NVENC | AMD AMF | Intel QSV | Intel VA-API | AMD VA-API |Apple VideoToolbox| Rockchip RKMPP | Software |
-| -------------- | ------------ | ------- | --------- | ------------ | ---------- | ---------------- | -------------- | -------- |
-| Windows        | ✔️           | ✔️      | ✔️        | N/A          | N/A        | N/A              | N/A           | WIP      |
-| Windows Docker | ✔️           | N/A     | N/A       | N/A          | N/A        | N/A              | N/A            | WIP      |
-| Linux          | ✔️           | ✔️      | ✔️        | ✔️           | ✔️        | N/A              | ✔️             | WIP      |
-| Linux Docker   | ✔️           | ✔️      | ✔️        | ✔️           | ✔️        | N/A              | ✔️             | WIP      |
-| macOS          | N/A          | N/A     | N/A       | N/A          | N/A        | ✔️               | N/A            | WIP      |
+| OS/Platform    | NVIDIA NVENC | AMD AMF | Intel QSV | Intel VA-API | AMD VA-API | Apple VideoToolbox | Rockchip RKMPP | Software |
```

Comment:
After looking at this again I think we can remove this table altogether? It does not make any sense to have a table like this now because every platform supports tone mapping on every possible hardware acceleration methods for that platform now and this table does not provide extra information.

Reply:
Sounds fair